src/app/common/controllers/FAQCtrl.js

```diff
@@ -0,0 +1,24 @@
+angular.module('just.common')
```

Comment:
Finns inget stöd för FAQ in APIt nu har sett det som en front end grej, men skulle det faktiskt kunna vara en bra idé att låta det styras därifrån.

Måste dock funderas lite på hur det ska "exponeras" så att det får stöd för flera språk på ett bra och enkelt sätt. Förslag och idéer uppskattas https://github.com/justarrived/just_match_api/issues/292 :)